app/src/common/utils/waitForSelector.js

```diff
@@ -0,0 +1,10 @@
+import { select, take } from 'redux-saga/effects';
+
+export function* waitForSelector(selector) {
```

Comment:
Could you please explain in more details what is the purpose of this generator?

Reply:
Since the project settings request does not call from launch page Saga, I cannot know when response got, the only indicator is the active filter selector, this generator waits for any valid selector value and is interrupted.